storybook/stories/API/cartesian/Area.stories.tsx

```diff
@@ -121,34 +205,20 @@ export const Stacked = {
             bottom: 20,
             left: 20,
           }}
-          data={data}
+          data={pageData}
         >
-          <Area stackId="pv-uv" dataKey={dataKey1} stroke={areaColor1} fill={areaColor1} {...areaArgs} />
-          <Area stackId="pv-uv" dataKey={dataKey2} stroke={areaColor2} fill={areaColor2} {...areaArgs} />
+          <Area fill="red" isAnimationActive={false} stackId={args.stackId1} dataKey="uv" />
+          <Area fill="green" isAnimationActive={false} stackId={args.stackId2} dataKey="pv" />
         </ComposedChart>
       </ResponsiveContainer>
     );
   },
   args: {
-    data: coordinateWithValueData,
-    dataKey1: 'x',
-    dataKey2: 'y',
-    areaColor1: 'lightblue',
-    areaColor2: 'lightgreen',
-    isAnimationActive: false,
-  },
-  argTypes: {
-    areaColor1: {
-      control: { type: 'color' },
-    },
-    areaColor2: {
-      control: { type: 'color' },
-    },
+    stackId1: '1',
+    stackId2: '1',
```

Comment:
Changed this story to actually expose the stackId and removed props that are otherwise shown in the Style story.